Client/src/Pages/Monitors/CreateNewMonitor.jsx

```diff
@@ -228,7 +257,14 @@ const CreateNewMonitor = () => {
         rightLayout={
           <div className="advanced-setting-config">
             <div className="adv-setting-title">Check frequency</div>
-            <FlexibileTextField />
+            <div className="monitors-gaps-small"></div>
+            <Dropdown
+              id="freq-dropdown"
+              label="1 minutes"
+              onChange={handleFrequencyOptionChange}
+              options={frequencyOptions}
```

Comment:
should this just be frequency instead of frequency options? Maybe that will fix the issue?

Reply:
No, apparently this is an array type that needs to be passed.